packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ipv6/ipv6.service.js

```diff
@@ -0,0 +1,28 @@
+import { PRODUCT_TYPES } from './ipv6.constant';
+
+export default class IpAgoraV6Order {
+  /* @ngInject */
+  constructor(iceberg) {
+    this.iceberg = iceberg;
+  }
+
+  getVrackService() {
+    return this.iceberg('/vrack')
+      .query()
+      .expand('CachedObjectList-Pages')
+      .execute()
+      .$promise.then(({ data }) => {
+        return [
+          ...data.map((vrackService) => {
```

Comment:
Why using spread operator here `return [...data.map()]` ?
`return data.map()` is enough